README.md

```diff
@@ -137,36 +137,22 @@ To use the popup login flow, you'll need a popup application running on a
 trusted domain which authenticates the user, handles redirects, and messages the
 authenticated session back to your application.
 
-Due to the possible redirects and the security model of `window.postMessage`,
-you'll need to build a static popup app bound to your application's origin.
+In order to tell the user they're logging into *your* app, you'll need to
+build a static popup bound to your application's name.
 
-Keeping this in mind, here's how to get things working.
+Keeping this in mind, it's pretty simple to build a popup for your app!
```

Comment:
I suppose they could even just edit the HTML at this point, it has become so easy :wink: